pysilcam/config_example.ini

```diff
@@ -22,6 +22,7 @@ pix_size = 28.758169934640524
 path_length = 12
 com_port = COM4
 window_size = 20
+img_crop = (500, 500, 1750, 1750)

```

Comment:
Since a new option is added, should the file version number be bumped?

Reply:
Done in commit.